.textlintrc

```diff
@@ -0,0 +1,63 @@
+{
+  "rules": {
+    "en-spell": {
+        "personalDictionary": [
+          "clj",
+          "crd",
+          "docker",
+          "github",
+          "golang",
+          "helm",
+          "homebrew",
+          "k3d",
+          "k3s",
+          "k8s",
+          "kubernetes",
+          "ngt",
+          "ngtd",
+          "onnx",
+          "operator",
+          "scalable",
+          "spotify",
+          "sptag",
+          "vald",
+          "vektor",
+          "wald",
+          "mevie",
+          "upsert",
+          "vectorize",
+          "grafana",
+          "jaeger",
+          "sdk",
+          "uuid",
+          "changelog",
+          "http",
+          "rerank",
+          "goimports",
+          "golangci",
+          "int32",
+          "uint32",
+          "fastime",
+          "idrequest",
+          "config",
+          "changelog",
+          "uuid",
+          "godoc",
+          "struct",
+          "goroutine",
+          "errgroup",
+          "subtest",
+          "unexported",
```

Comment:
:memo: I have added this words.

Reply:
Can you sort each words?

Reply:
Thank you for your comment.
I have sorted these words: https://github.com/vdaas/vald/pull/1715/commits/a855e631891ddbc092d0327e03633d879b278032